jumpscale/entry_points/threebot.py

```diff
@@ -248,21 +248,9 @@ def cli():
 
 def have_wallets():
     wallets = j.clients.stellar.list_all()
-    test, main = False, False
     for wallet_name in wallets:
         wallet = j.clients.stellar.get(wallet_name)
-        if wallet.network.value == "TEST":
-            test = True
-        elif wallet.network.value == "STD":
-            main = True
-    return test, main
-
-
-def create_test_wallet(wallet_name):
-    try:
-        j.clients.stellar.create_testnet_funded_wallet(wallet_name)
-    except Exception as e:
-        j.logger.error(str(e))
+        return wallet.network.value == "STD"
```

Comment:
this is an early return .. should return True in case of the first one, but also not return False immediately before exhausting the loop